### DISCUSSION TITLE
Lens Profiles

### DISCUSSION THREAD

**Discussion by numiromeneses** [latenitefilms/GyroflowToolbox#21] [Support] [answered]
Hello,
I'm finding it difficult to create a lens profile.
I mean, the process is okey, but there is probalby something I'm not doing quite right, because when I use it on Final Cut it gets messy.
Can someone help?
I'm using canon and laowa lenses on blackmagic ccp4k and 6k.

Thanks

**Comment by latenitefilms** [comment, marked as answer]
Gyroflow Toolbox uses Gyroflow as its "render engine" so, in theory, whatever you see in Gyroflow should be identical in Gyroflow Toolbox - so I'm not really sure what you mean by "gets messy"?

There's lots of useful information in the Gyroflow Documentation site:

https://docs.gyroflow.xyz/app/getting-started/lens-calibration

If you need help with Gyroflow, their Discord Community is also very active and helpful:

https://discord.com/invite/BBJ2UVAr2D
